README.adoc

```diff
@@ -15,7 +11,7 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
+* This is a desktop AlgoBase application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
 * It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
```

Comment:
Probably we can have some brief introduction to AlgoBase instead?

Reply:
Ok I'll merge this with the intro for our UG